echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/googlechat/GoogleChatService.java

```diff
@@ -27,7 +27,7 @@ public GoogleChatService(GoogleChatClient googleChatClient) {
     this.googleChatClient = googleChatClient;
   }
 
-  Response sendMessage(String webhook, GoogleChatMessage message) {
-    return googleChatClient.sendMessage(webhook, message);
+  void sendMessage(String webhook, GoogleChatMessage message) {
```

Comment:
Why change the signature here?  Doesn't the caller care about the response?  If not that seems like something to clean up in another PR.